src/plonkish/ir/sc.rs

```diff
@@ -29,6 +31,30 @@ impl<F, MappingArgs> SuperCircuit<F, MappingArgs> {
     pub fn get_mapping(&self) -> MappingGenerator<F, MappingArgs> {
         self.mapping.clone()
     }
+
```

Comment:
@qwang98   The IR from plonklish should not be used in PIL

Reply:
I agree. I appended the sub circuit asts (now SBPIR) to the super circuit IR and that's why I took our IR as an input (so that I can access the sub circui asts appended). I think this is a quite clumsy work around and I'll instead take the sub circuit asts directly.